lib/writer2.c

```diff
@@ -180,7 +180,7 @@ appendLibId(ktxHashList* head, ktxHashListEntry* writerEntry)
  * @param[in] This      pointer to the target ktxTexture object.
  * @param[in] level     mip level of the image to set.
  * @param[in] layer     array layer of the image to set.
- * @param[in] faceSlice cube map face or depth slice of the image to set.
+ * @param[in] faceSlice cube map face or depth slice of the image to set or (-1) to set the entire level.
```

Comment:
Make this and other similar lines <= 80 characters. Going on to a second line is okay.

Reply:
done